src/common/helpers.js

```diff
@@ -5,3 +5,5 @@ export const bindMethods = (context, methods) => {
 };
 
 export const noop = Function.prototype;
+
+export const KEY_CODES = { TAB_KEY: 9, ENTER_KEY: 13 };
```

Comment:
Is it really common?
If not, it should live under `AutoComplete/constants.js`

tbh, I'm not sure but let agree the first one outside of the autocomplete scope that wants to use it, will just move it into `common/helpers`?

Check out how the Alert is using inner helpers and constants:
https://github.com/patternfly/patternfly-react/tree/master/src/components/Alert

Reply:
Great point. Thoughts @priley86?

Reply:
keyboard key codes will be used very often when implementing a11y from a JS standpoint. it's mostly about handling enter press/escape press/tab press from the JS side...so for that reason, i would propose keeping it common to all.